core/assignment.go

```diff
@@ -2,9 +2,24 @@ package core
 
 import (
 	"errors"
+	"fmt"
+	"math"
 	"math/big"
 )
 
+const (
+	percentMultiplier    = 100
```

Comment:
What is percentMultiplier and why do we need this extra parameter?

Reply:
Used when we need to do `QuorumThreshold-AdversaryThreshold/100` to avoid magic numbers/make the code more readable.